src/components/NavBar/NavBar.tsx

```diff
@@ -125,10 +123,10 @@ const NavBar: React.FunctionComponent = () => {
           Commander we hope you are ready for today&apos;s daily chellenge
         </Modal.Body>
         <Modal.Footer>
-          <Button variant="secondary" onClick={handleCloseAvailable}>
+          <Button className={styles.dcmodalBtn} onClick={handleCloseAvailable}>
```

Comment:
Refer to [react-bootstrap](https://react-bootstrap.github.io/components/buttons/)
```suggestion
<Button variant="outline-danger" onClick={handleCloseAvailable}>
```